.github/workflows/sdks.yaml

```
@@ -0,0 +1,20 @@
+name: SDKs
+
+on: [push]
```

This won't trigger build for pull requests. Use the following:
```suggestion
on:
  pull_request:
  push:
    branches:
      - main
```